packages/compartment-mapper/src/assemble.js

```diff
@@ -276,8 +282,8 @@ export const assemble = (
     const {
       location,
       modules = {},
-      parsers = {},
-      types = {},
+      parsers: languageForExtension = {},
+      types: languageForModuleSpecifier = {},
```

Comment:
Would it make sense to rename the property names to match these much clearer variable names?

Reply:
It might. They’re very verbose for a JSON file, but it might be a good idea.